extra/html-extension/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "twig/html-extension",
+    "type": "library",
+    "description": "A Twig extension for HTML",
+    "keywords": ["twig", "html"],
+    "homepage": "https://twig.symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com",
+            "homepage": "http://fabien.potencier.org",
+            "role": "Lead Developer"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "twig/twig": "^2.4|^3.0"
+    },
+    "require-dev": {
+        "symfony/mime": "^4.3",
+        "symfony/phpunit-bridge": "^4.4@dev"
```

Comment:
I think this should be moved to "require" or?

Reply:
The mime component is "only" used for 1 filter (not the other) and only if you don't pass the content-type. I hesitated on this one. Might be "better" to always require it.

Reply:
I would try to get rid of optional dependencies.

Reply:
And installing it when it's not needed does seem better than not installing it, when it is needed.